homework/index.js

```diff
@@ -30,18 +32,129 @@
+      const tr1 = createAndAppend('tr', tbody);
+      createAndAppend('td', tr1, { class: 'label', text: 'Repository :' });
+      const td2 = createAndAppend('td', tr1);
+      createAndAppend('a', td2, {
+        href: selectedRepository.html_url,
+        target: '_blank',
+        text: selectedRepository.name,
+      });
+    }
+    if (selectedRepository.description !== null) {
+      const tr2 = createAndAppend('tr', tbody);
+      createAndAppend('td', tr2, { class: 'label', text: 'Description :' });
+      const repositoryDescription = createAndAppend('td', tr2, { text: '' });
+      repositoryDescription.textContent = selectedRepository.description;
+    }
+    if (selectedRepository.forks !== null) {
+      const tr3 = createAndAppend('tr', tbody);
+      createAndAppend('td', tr3, { class: 'label', text: 'Forks :' });
+      const fork = createAndAppend('td', tr3, { text: '' });
+      fork.textContent = selectedRepository.forks;
+    }
+    if (selectedRepository.updated_at !== null) {
+      const tr4 = createAndAppend('tr', tbody);
+      createAndAppend('td', tr4, { class: 'label', text: 'Updated :' });
+      const updated = createAndAppend('td', tr4, { text: '' });
+      updated.textContent = new Date(selectedRepository.updated_at).toLocaleString();
+    }
+  }
+
+  function clearContainer(container) {
+    while (container.hasChildNodes()) {
+      container.removeChild(container.firstChild);
+    }
+  }
+
+  function createContributions(contributors) {
+    const container = document.getElementById('container');
+    const rightDiv = createAndAppend('div', container, {
+      id: 'rightDiv',
+      class: 'right-div white_Frame',
+    });
+    createAndAppend('p', rightDiv, {
+      class: 'contributor-header',
+      text: 'Contributions :',
     });
+    const ul = createAndAppend('ul', rightDiv, { id: 'ulList', class: 'contributor-list' });
+    contributors.forEach(element => {
+      const li = createAndAppend('li', ul, {
+        class: 'contributor-item',
+        tabindex: 0,
+      });
+      createAndAppend('img', li, {
+        src: element.avatar_url,
+        height: 48,
+        class: 'contributor-avatar',
+        alt: 'contributor',
+      });
+      const contInfoDiv = createAndAppend('div', li, { class: 'contributor-data' });
+      createAndAppend('div', contInfoDiv, { text: element.login });
+      createAndAppend('div', contInfoDiv, {
+        class: 'contributor-badge',
+        text: element.contributions,
+      });
+      li.addEventListener('click', () => {
+        window.open(element.html_url, '_blank');
+      });
+    });
+  }
+
+  function listenSelectElement(repositories, root) {
+    const container = document.getElementById('container');
+    const select = document.getElementById('select');
+    select.addEventListener('change', () => {
+      clearContainer(container);
+      const newSelectedRepository = repositories[select.value];
+      createContainer(newSelectedRepository);
```

Comment:
This code to the end of the function is *exactly* the same as that used in create Header.
I would have refactored this code into it's own function e.g. `setContent` or something.  And just called that from those two places.
It is not essential, but will make maintaining the code easier.